### ISSUE TITLE
getting TLS handshake error.

### ISSUE THREAD

**Issue by thamilselvam-source** [terraform-google-modules/terraform-google-kubernetes-engine#1911]
### TL;DR

I was using the acm module, but when trying to create git_creds and other annotations like gate keeper, it was throws an error as http/TLS handshake error.

### Additional information

Error: Get "
https://34.166.70.21/api?timeout=32s":
net/http: TLS handshake timeout
│
│   with module.acm.kubernetes_annotations.annotate-sa-config-management-monitoring[0],
│   on ..\..\Modules\acm\creds.tf line 53, in resource "kubernetes_annotations" "annotate-sa-config-management-monitoring":    
│   53: resource "kubernetes_annotations" "annotate-sa-config-management-monitoring" {
│
╵
╷
│ Error: Get "
[https://34.166.70.21/api?timeout=32s":](https://34.166.70.21/api?timeout=32s%22:)
net/http: TLS handshake timeout
│
│   with module.acm.kubernetes_annotations.annotate-sa-gatekeeper-system[0],
│   on ..\..\Modules\acm\creds.tf line 71, in resource "kubernetes_annotations" "annotate-sa-gatekeeper-system":
│   71: resource "kubernetes_annotations" "annotate-sa-gatekeeper-system" {
│
╵
╷
│ Error: Post "
[https://34.166.70.21/api/v1/namespaces/config-management-system/secrets":](https://34.166.70.21/api/v1/namespaces/config-management-system/secrets%22:)
net/http: TLS handshake timeout       
│
│   with module.acm.kubernetes_secret_v1.creds[0],
│   on ..\..\Modules\acm\creds.tf line 120, in resource "kubernetes_secret_v1" "creds":
│  120: resource "kubernetes_secret_v1" "creds" {
│
